README.md

```diff
@@ -1,60 +1,87 @@
-# Which is the fastest?
+# Which is the fastest ?
+----------
+#### Simple framework comparison
+----------
+<p align="center">
+   <a href="https://github.com/the-benchmarker/web-frameworks/actions?query=workflow%3ACI" target="_blank">
+      <img src="https://github.com/the-benchmarker/web-frameworks/workflows/CI/badge.svg" alt="Test">
+   </a>
+   <a href="https://join.slack.com/t/thebenchmarker/shared_invite/zt-fcyy1ybq-A7T1SedewiVMEtJQGEyQYw" target="_blank">
+      <img src="https://img.shields.io/badge/slack-chat_with_us-green" alt="Chat with us">
+   </a>
+   <a href="https://github.com/the-benchmarker/web-frameworks/blob/master/LICENSE" target="_blank">
+      <img src="https://img.shields.io/github/license/the-benchmarker/web-frameworks" alt="License">
+   </a>
+</p>
 
-![CI](https://github.com/the-benchmarker/web-frameworks/workflows/CI/badge.svg)
+## Motivation
 
-[![Chat with US](https://img.shields.io/badge/slack-Chat_with_us-blueviolet)](https://thebenchmarker.slack.com)
+It exists plenty of frameworks. Each one has its advantages, and drawbacks. The idea of this project is to list those frameworks and to expose metrics for each one listed `here` (performance is only the first one)
 
-This project aims to be a load benchmarking suite, no more, no less
+#### What is a framewok ?
 
-> Measuring response times (routing times) for each framework (middleware).
+A framework is a set of components working together. There is a **huge** variety of frameworks, from full-stack (when deling with multiple layers) to micro (when providing only routing part)
```

Comment:
"dealing"

I am not sure that this is a sufficient definition.

Reply:
yeah need probably more details, but try to keep it simple

will take time to clarify